lib/utils/parse-options.js

```diff
@@ -31,10 +35,19 @@ function parseOptions (options) {
     options.cache.timeout = DEFAULTS.cache.timeout
   }
 
+  if (options.interval && typeof options.interval === 'number') {
```

Comment:
I think we could simply do just `if (typeof options.interval === 'number')` here, right?